pkg/limayaml/limayaml.go

```diff
@@ -10,6 +10,7 @@ type LimaYAML struct {
 	VMType             *VMType       `yaml:"vmType,omitempty" json:"vmType,omitempty"`
 	OS                 *OS           `yaml:"os,omitempty" json:"os,omitempty"`
 	Arch               *Arch         `yaml:"arch,omitempty" json:"arch,omitempty"`
+	Image              string        `yaml:"image,omitempty" json:"image,omitempty"`
 	Images             []Image       `yaml:"images" json:"images"` // REQUIRED
```

Comment:
It is confusing that `Image` (field name) is not `Image` (struct name) while `Images` is `[]Image`

Reply:
It's not perfect, the current solution was just "matching" what you would find in a Kubernetes yaml

Reply:
Another approach would be to just re-use the images field, similar to what we are doing with disk